### PR TITLE
2 tic 1

### LINKED ISSUE #2: TIC-1
implement feature 1

### PR DESCRIPTION
**Pull request by rahulsahani-wmp**
added rahullllll in home.html

### CODE REVIEW

**Review by Aanchal-Saxena — COMMENTED** [review]
nice code Rahul, good work

File: Home.html [hunk 1]
```diff
@@ -10,6 +10,9 @@
         <p>
         This is a paragraph segment. Here comes some content about something. This
         paragraph can be about anything.
+
+        rahullllllll
```
Comment (Aanchal-Saxena) [inline Home.html:14]: tooo many llllllll in rahul

File: Home.html [hunk 2]
```diff
@@ -10,6 +10,9 @@
...
+       
         </p>
         <h2>hey this is khushi, im changing things on the new branch.</h2>
```
Comment (Aanchal-Saxena) [inline Home.html:17]: nice paragraph

**Review by Aanchal-Saxena — APPROVED** [review]

File: Home.html (hunk 2 above)
Comment (rahulsahani-wmp) [reply Home.html:17]: thanku

File: Home.html [hunk 3]
```diff
@@ -10,6 +10,9 @@
         <p>
         This is a paragraph segment. Here comes some content about something. This
```
Comment (Khushi006) [inline Home.html:11]: this is very commonly used sentence use something new

**Pull request merged**